app/controllers/orgs/rosters_controller/lti_dependency.rb

```diff
@@ -17,10 +17,17 @@ def import_from_lms
         "User IDs": students.map(&:user_id),
         "Names": students.map(&:name),
         "Emails": students.map(&:email)
-      }
+      }.select { |_, v| v.any? }
 
       GitHubClassroom.statsd.increment("lti_configuration.import")
 
+      if @identifiers.empty?
+        msg = "GitHub Classroom received an empty roster from your Learning Management System. Please ensure you
+        have enabled GitHub Classroom to receive student information from within your Learning Management System."
```

Comment:
Are we able to reword this error to be more actionable? WHat does "enable GitHub Classroom to receive student information" mean? Do they need to enable the membership service?

Reply:
Yeah, instead of an error message here I was instead thinking of actually changing the view so that instead of a select list on an empty response, we explained in more detail what the problem was and later possibly link them to support docs.

The error here isn't that the membership service isn't enabled or the request failed (in fact this screen will be reached only when it receives a successful response from the membership service), it's that the membership service _is_ enabled but the teacher set the privacy settings (or they are that way by default) such that individual student information isn't being sent in the roster, so it's essentially an empty response.

Reply:
Gotcha. If you'd like to ship this as a V1 then iterate, that sounds :+1:, but :100: to explaining the error in more detail :)